soc/arm/nordic_nrf/nrf51/Kconfig.series

```diff
@@ -9,6 +9,7 @@ config SOC_SERIES_NRF51X
 	select ARM
 	select CPU_CORTEX_M0
 	select SOC_FAMILY_NRF
+	select NRF_RTC_TIMER
```

Comment:
How does someone opt out of this and use e.g. the cortex M systick instead?

Reply:
Applies suggestion from https://github.com/zephyrproject-rtos/zephyr/pull/67446#discussion_r1452025576